.github/workflows/lint-test.yml

```diff
@@ -23,13 +23,20 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Setup Poetry 🔧
+        run: |
```

Comment:
Lets setup poetry as we do in the other repos:
https://github.com/ThePalaceProject/library-registry/blob/main/.github/workflows/test-build.yml#L27-L28